stl/inc/format

```diff
@@ -1428,10 +1538,18 @@ _NODISCARD _OutputIt _Write_aligned(_OutputIt _Out, const int _Width, const _Bas
         }
     }
 
-    // TRANSITION, add support for unicode/wide formats
-    _Out = _RANGES fill_n(_STD move(_Out), _Fill_left, _Specs._Fill[0]);
+    const basic_string_view<_CharT> _Fill_char{_Specs._Fill, _RANGES find(_Specs._Fill, '\0')};

```

Comment:
I belive this is broken

1. We should use `_RANGES _Unchecked_find`
2. If we have wide characters this is only one element, so we should not search
3. If we have plain characters you just changed `_Fill` to only contain _CharType(` `) so you will always find `end(_Specs._Fill)`

Reply:
we might want to just store the length in character types of the fill character array

Reply:
1. I'm too lazy to call `begin` and `end` on a range when the range algorithm does so for me.
2. Wide characters are UTF-16 on our implementation, so two may be needed to represent a fill character (the test case that uses 🏈, for example).
3. `meow woof[4] = {quack};` is equivalent to `meow woof[4] = {quack, {}, {}, {}};`, not to `meow woof[4] = {quack, quack, quack, quack};`.